pages/index.tsx

```diff
@@ -1,10 +1,22 @@
+import fs from "fs";
+import path from "path";
 import "styled-jsx";
 
 import About from "../components/about";
+import { getSVGMap } from "../components/image_generation/common";
 import Mint from "../components/mint";
 import TheSantas from "../components/the-santas";
 
-export default function Home() {
+export function getStaticProps() {
+  return {
+    props: {
+      // @NOTE: this has to be done server side to leverage fs
```

Comment:
Is that already done here? How do you configure something as server side vs client?